smoketest/migrations_test.go

```diff
@@ -667,6 +667,7 @@ func TestMigrations(t *testing.T) {
 			if checkDiff(t, "UP", migrationName, upSnap, s) {
 				t.Fatalf("UP migration did not restore previous schema")
 			}
+			t.Parallel()
```

Comment:
Migration tests cannot be run in parallel without each having an independent harness/DB. Which should probably wait for the refactor.